AGENTS.md

```
@@ -174,11 +174,77 @@ report that you bounded it. Mutate the halves separately; they fail differently
+claim. It was not: **the PR came from an outside contributor**, who has no
+reason to know or care about an internal assignment, and for whom the repo's
+existing rules already apply (look for their PR before starting, external work
+takes precedence, never push to their branch).
+
+The session that self-assigned never ran `gh pr list --search 3012`. No
+assignee field, however precise about sessions, would have helped: **no session
+held it.** Only the PR search would have.
+
+So run both, every time:
+
+    gh issue view <n> --json assignees      # is one of us on it
```

markdownlint-cli2 (0.23.2)

[warning] Code block style
Expected: fenced; Actual: indented

(MD046, code-block-style)